WordPress/src/main/java/org/wordpress/android/util/config/RemoteConfigField.kt

```diff
@@ -1,13 +1,13 @@
 package org.wordpress.android.util.config
 
-open class RemoteConfigField<T>(val appConfig: AppConfig, val remoteField: String, val defaultValue: T) {
+open class RemoteConfigField<T>(val appConfig: AppConfig, val remoteField: String) {
```

Comment:
❓ Do we still need the generic `T` after the changes to this class?

Reply:
We technically don't but I think it adds a good indicator of what the class expects, so I just pushed a change making sure it is used in `getValue`.